src/Pages/ProductDetail/ProductDetail.jsx

```diff
@@ -1,8 +1,191 @@
 import React, { Component } from 'react';
-
+import './ProductDetail.scss';
+import Review from './Components/Review';
 class ProductDetail extends Component {
+  state = {
+    count: 0,
+    addClass: false,
+    show: false,
+  };
+
+  handleClick = e => {
+    this.setState({ addClass: !this.state.addClass });
+  };
+
+  handleIncrease = () => {
+    const count = this.state.count + 1;
+    this.setState({ count: count < 20 ? count : 20 });
+  };
+  handleDecrease = () => {
+    const count = this.state.count - 1;
+    this.setState({ count: count < 0 ? 0 : count });
+  };
   render() {
-    return <div>ProductDetail</div>;
+    return (
+      <>
+        <section className="ProductDetail">
+          <div className="productInfoContainer">
+            <div className="productImg">
+              <img src="../images/cutedog.png" alt="" className="bigImg" />
+              <div className="smallImgs">
+                <img
+                  src="../images/pets-4383147_640.jpg
+                "
+                  alt=""
+                  className="smallImg"
+                />
+              </div>
+            </div>
+            <form
+              onSubmit={e => {
+                e.preventDefault();
+              }}
+            >
+              <h2>PUPPY FOOD</h2>
+              <h1 className="infoTitle">
+                Turkey with Duck, Sweet Potato, Dill & Camomile (Puppy)
+              </h1>
+              <span className="priceInfo">$15.00</span>
+              <div className="selectorWraper">
+                <div
+                  onClick={this.handleClick}
+                  className={`${
+                    !this.state.addClass ? 'selected' : 'non-selected'
+                  }`}
```

Comment:
```suggestion
                  className={
                    !this.state.addClass ? 'selected' : 'non-selected'
                  }
```

- 구조 분해 할당 전부 적용해주시고, 아래에도 같은 내용 있는데 같이 수정 부탁드립니다!